api/src/main/java/org/pmiops/workbench/user/DevUserRegistrationService.java

```diff
@@ -0,0 +1,8 @@
+package org.pmiops.workbench.user;
+
+import com.google.api.services.oauth2.model.Userinfoplus;
+import org.pmiops.workbench.db.model.DbUser;
+
+public interface DevUserRegistrationService {
+  DbUser createUserFromUserInfo(Userinfoplus userInfo);
```

Comment:
Might as well use the full class name `createUserFromUserinfoplus` or leave it out, i.e. `createUser()`.

Reply:
Yeah, I agree this isn't super consistent. Just `createUser` seems like an all around improvement – done.